### PR TITLE
contribution guidelines checks

### PR DESCRIPTION
**Pull request by matkosoric**
### **Describe your change:**

documentation change - spelling

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: machine_learning/linear_regression.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 Linear regression is the most basic type of regression commonly used for
-predictive analysis. The idea is pretty simple, we have a dataset and we have
-a feature's associated with it. The Features should be choose very cautiously
-as they determine, how much our model will be able to make future predictions.
-We try to set these Feature weights, over many iterations, so that they best
-fits our dataset. In this particular code, i had used a CSGO dataset (ADR vs
+predictive analysis. The idea is pretty simple: we have a dataset and we have
+features associated with it. Features should be chosen very cautiously
+as they determine how much our model will be able to make future predictions.
+We try to set these features weights, over many iterations, so that they best
```
Comment (cclauss) [inline machine_learning/linear_regression.py]: We try to set the weight of these features...

File: sorts/comb_sort.py [hunk 2]
```diff
@@ -15,40 +20,43 @@
...
 
-    while gap > 1 or swapped:
-        # Update the gap value for a next comb
-        gap = int(float(gap) / shrink_factor)
+    while not completed:
 
-        swapped = False
-        i = 0
+        # Update the gap value for a next comb
+        gap = int(gap / shrink_factor)
```
Comment (cclauss) [inline sorts/comb_sort.py]: gap //= shrink_factor  # will give us integer division

File: sorts/comb_sort.py [hunk 3]
```diff
@@ -1,8 +1,13 @@
 """
+This is pure python implementation of comb sort algorithm.
```
Comment (cclauss) [inline sorts/comb_sort.py]: python --> Python  # it is a brand name, not a snake.

Comment (TheSuperNoob) [reply sorts/comb_sort.py]: Is it even needed to say its implemented in Python given that this entire repository should just be python algorithms?

Comment (cclauss) [reply sorts/comb_sort.py]: I think it is fine to write that this is a Python implementation.
The code might be copied elsewhere outside this repo.

File: sorts/comb_sort.py (hunk 2 above)
Comment (matkosoric) [reply sorts/comb_sort.py]: does not work, return type is still float

Comment (cclauss) [reply sorts/comb_sort.py]: You are correct.  Sorry for the mistake.
```python
>>> 35.2 / 7
5.028571428571429
>>> 35.2 // 7
5.0
```

File: sorts/comb_sort.py [hunk 4]
```diff
@@ -13,42 +18,45 @@
...
+    :param data: mutable collection with comparable items
+    :return: the same collection in ascending order
     Examples:
     >>> comb_sort([0, 5, 3, 2, 2])
     [0, 2, 2, 3, 5]
     >>> comb_sort([])
     []
-    >>> comb_sort([-2, -5, -45])
-    [-45, -5, -2]
+    >>> comb_sort([99,45,-7,8,2,0,-15,3])
```
Comment (cclauss) [inline sorts/comb_sort.py]: Comma space please.

**Comment by cclauss** [comment]
NICE!!!

**Pull request merged**